test/helpers/helpers.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs');
 const os = require('os');
 const process = require('process');
 const StatsD = require('../../lib/statsd.js');
-
+const { PROTOCOL: { TCP, UDP, UDS } } = require('../../lib/constants');
```

Comment:
I'd personally prefer `const { TCP, UDP, UDS } = require('../../lib/constants').PROTOCOL;` - I find multi-layered unwrapping needlessly hard to follow. KISS and all...